train-tracker/src/Search.js

```diff
@@ -0,0 +1,19 @@
+import './styles/Search.css';
+
+function Search({searchChange, selectChange, searchVal, searchByVal}){
+    return (
+        <form className='form'>
+                <label>
+                    Search by: 
+                    <select onChange={selectChange} value={searchByVal}>
+                        <option value="1">Train Number</option>
+                        <option value="2">Line Name</option>
+                    </select>
+                </label>
+                <input onChange={searchChange} type="text" value={searchVal}></input>
```

Comment:
What do you mean by this? Search is a controlled component so the value needs to be the App state passed down through props, but I could change the default value?

Reply:
instead of `value={searchVal}` you could do `placeholder={searchVal}`, notice the grayed out text instead of needing to overwrite the "enter train line or number." Search seems to still work in my testing

https://www.w3schools.com/tags/att_input_placeholder.asp

![image](https://github.com/user-attachments/assets/e5797eac-51ac-4c9f-a7c3-0c98bab318a4)

Reply:
Awesome sorry I was confused by what you meant, will add this